pkgs/games/hexgui/default.nix

```diff
@@ -0,0 +1,34 @@
+{ ant
+, fetchFromGitHub
+, jdk
+, lib
+, makeWrapper
+, stdenv
+}:
+stdenv.mkDerivation {
+  pname = "hexgui";
+  version = "unstable-2022-5-30";
+
+  src = fetchFromGitHub {
+    owner = "selinger";
+    repo = "hexgui";
+    rev = "d94ce1d35a22dad28d3e7def4d28e6bebd54da9d";
+    hash = "sha256-1MroFH2JSEZHFigcsw1+xyHJWEnHTvHmRPVirUgwM6I=";
+  };
+
+  nativeBuildInputs = [ ant jdk makeWrapper ];
```

Comment:
Why do we have jdk in nativeBuildInputs?

Reply:
Needed for `ant` I believe.

Reply:
I just copied this https://nixos.org/manual/nixpkgs/stable/#sec-language-java